dist/algoliasearch.helper.js

```diff
@@ -10623,8 +10623,8 @@ var lib = {
    * @param {string} value the value of the refinement
    * @return {RefinementList} a new and updated list
    */
-  toggleRefinement: function toggleRefinement(refinementList, attribute, value) {
```

Comment:
Which version will deprecate it? I picked 2.19.0 now since that's the next major.